posthog/hogql_queries/error_tracking_query_runner.py

```diff
@@ -231,12 +152,15 @@ def calculate(self):
         )
 
     def results(self, columns: list[str], query_results: list):
-        mapped_results = [dict(zip(columns, value)) for value in query_results]
         results = []
+        mapped_results = [dict(zip(columns, value)) for value in query_results]
+
+        issue_ids = [result["id"] for result in mapped_results]
+        issues = self.error_tracking_issues(issue_ids)
+
         for result_dict in mapped_results:
-            fingerprint = self.query.fingerprint if self.query.fingerprint else result_dict["fingerprint"]
-            group = self.group_or_default(fingerprint)
-            results.append(result_dict | group)
+            issue = issues.get(result_dict["id"])
+            results.append(issue | result_dict | {"assignee": self.query.assignee})
```

Comment:
This assumes we never delete an issue row in PG - right? Seems safe enough but just wanted to double check

Reply:
Yes. I don't think we should hard delete issues, especially given we have the "archived" state

Reply:
It also assumes we never have an event with no associated issue I think, which won't be true so long as we support the sentry integration :/